connectors/src/styles/globals.css

```diff
@@ -0,0 +1,16 @@
+html,
```

Comment:
We might need it in the future, so should we keep it as an example of how to implement it or just remove it and figure out later?